x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -1573,16 +1547,33 @@ describe('Task Runner', () => {
           },
         ],
       });
+
+      alertsService.createAlertsClient.mockImplementation(() => {
+        return {
+          initializeExecution: jest.fn(),
+          processAndLogAlerts: jest.fn(),
+          getTrackedAlerts: jest.fn(),
+          getProcessedAlerts: jest.fn(),
+          getAlertsToSerialize: jest.fn().mockResolvedValueOnce({ state: {}, meta: {} }),
+          hasReachedAlertLimit: jest.fn(),
+          checkLimitUsage: jest.fn(),
+          persistAlerts: jest.fn(),
+          getSummarizedAlerts: jest.fn().mockResolvedValue({
+            new: {
+              count: 1,
+              data: [mockAAD],
+            },
+            ongoing: { count: 0, data: [] },
+            recovered: { count: 0, data: [] },
+          }),
+          factory: jest.fn(),
+          client: jest.fn(),
+        };
+      });
+
       encryptedSavedObjectsClient.getDecryptedAsInternalUser.mockResolvedValue(mockedRawRuleSO);
       const result = await taskRunner.run();
 
```

Comment:
should we replace this expect with an expect that `mockAlertsClient.getSummarizedAlerts` was called?

Reply:
added back